_api-reference/cluster-health.md

```diff
@@ -70,6 +80,28 @@ wait_for_status | Enum | Wait until the cluster health reaches the specified sta
 }
 ```
 
+## Response fields
+
+The following table lists all response fields.
+
+|Field	|Data type	|Description	|
+|:---	|:---	|:---	|
+|cluster_name	| String	| The name of the cluster.	|
+|status	| String	| The health status of the cluster that represents the health of its primary and replica shards.	The status may be `green`, `yellow`, or `red`.|
```

Comment:
This sentence is confusing because it implies that there is a "shard health" metric, and that cluster health is just an alias for it.

What about simplifying it to something like:

```
Describes the state of shard allocation in the cluster. May be `green`, `yellow`, or `red`.
```

The way it reads right now makes it sound like cluster health is an alias for a metric called shard health.

Reply:
I want to mention "health status", so how about ```The cluster health status that represents the state of shard allocation in the cluster.```?